File: Tests/TestData/MultiFileCharacterization/SourceFiles/VbLibrary/ImportCases/ModuleImportClass.vb
Imports VbNetStandardLib.ModuleWithClassAndMethod

Namespace ModuleImport
    Class ModuleImportClass
        Public Sub ModuleImportUnQualifiedMember()
            ModuleMethod()
        End Sub

        Public Sub ModuleImportUnQualifiedNestedType()
            Dim mc = New ModuleClass()
            mc.ModuleClassMethod()
        End Sub
    End Class
End Namespace

File: Tests/TestData/MultiFileCharacterization/VBToCSResults/ConvertWholeSolution/VbLibrary/ImportCases/ModuleImportClass.cs
using static VbNetStandardLib.ModuleWithClassAndMethod;

namespace VbLibrary.ModuleImport
{
    class ModuleImportClass
    {
        public void ModuleImportUnQualifiedMember()
        {
            ModuleMethod();
        }

        public void ModuleImportUnQualifiedNestedType()
        {
            var mc = new ModuleClass();
            mc.ModuleClassMethod();
        }
    }
}